functests/1_performance/cpu_management.go

```diff
@@ -91,16 +93,50 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			Expect(conf).To(MatchRegexp(fmt.Sprintf(`"reservedSystemCPUs": ?"%s"`, reservedCPU)))
 
 			By("checking CPU affinity mask for kernel scheduler")
-			cmd = []string{"/bin/bash", "-c", "taskset -pc $(pgrep 'rcu_sched|rcu_preempt')"}
+			cmd = []string{"/bin/bash", "-c", "taskset -pc 1"}
 			sched, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
 			Expect(err).ToNot(HaveOccurred(), "failed to execute taskset")
 			mask := strings.SplitAfter(sched, " ")
 			maskSet, err := cpuset.Parse(mask[len(mask)-1])
 			Expect(err).ToNot(HaveOccurred())
 
-			Expect(reservedCPUSet.IsSubsetOf(maskSet)).To(Equal(true))
+			Expect(reservedCPUSet.IsSubsetOf(maskSet)).To(Equal(true), "The init process (pid 1) should have right cpu affinity mask")
```

Comment:
What do you mean "right affinity mask" ?

Reply:
heh, yeah - let's update it with values :)

Reply:
cool, thanks